examples/plot-utils-svm.jl

```diff
@@ -0,0 +1,31 @@
+import PGFPlots
+
+function plot_svm(outstruct, sol, name="tr-qr")
+    Comp_pg = outstruct.solver_specific[:SubsolverCounter]
+    objdec = outstruct.solver_specific[:Fhist] + outstruct.solver_specific[:Hhist]
+    x = outstruct.solution
+    a = PGFPlots.Axis(
+        [
+            PGFPlots.Plots.MatrixPlot(reshape(x, 28, 28);
+            colormap = PGFPlots.ColorMaps.GrayMap())#, zmin = -700, zmax = 700)#, legendentry="computed"),
```

Comment:
Is it ok if I leave this one? In case we want to set the axis manually?

Reply:
uncomment what you use. add arguments to the function if you want to be able to change the functionality